settings/base.py

```diff
@@ -32,6 +32,7 @@
 INSTALLED_APPS = [
     "landingpage",
     "dashboard",
+    "changepass",
```

Comment:
Having "changepass.apps.ChangepassConfig" (whatever is the proper name for the app config) will help cover the apps.py for changepass applict.

Reply:
done